### PR TITLE
Fix DataOStream compression instrumentation

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: co/dataOStream.cpp [hunk 1]
```diff
@@ -40,14 +41,15 @@ namespace co
 {
 namespace
 {
-//#define CO_INSTRUMENT_DATAOSTREAM
+#define CO_INSTRUMENT_DATAOSTREAM
```
Comment (tribal-tec) [inline co/dataOStream.cpp]: on by default now?

File: co/dataOStream.h [hunk 2]
```diff
@@ -233,8 +236,6 @@ class DataOStream : public boost::noncopyable
     /** Send the trailing data (command) to the receivers */
     void _sendFooter( const void* buffer, const uint64_t size );
 };
-
-std::ostream& operator << ( std::ostream&, const DataOStream& );
```
Comment (tribal-tec) [inline co/dataOStream.h]: why no more operator?

Comment (eile) [inline co/dataOStream.h]: Replaced by printStatistics(), which is more explicit on what it does.

**Comment by tribal-tec** [comment]
+1

**Pull request merged**